backend/endpoints/document_order.py

```
@@ -0,0 +1,80 @@
+@require_member_access()
+def add_document():
+    db = connect.get_db()
+    api = connect.get_api(db)
+
+    new_document_id = connect.get_body_arg("newDocumentId")
+    new_path = DocumentPath(new_document_id)
+    selected_document_id = connect.get_optional_body_arg("selectedDocumentId")
+
+    try:
+        document_name = get_document(api, new_path)["name"]
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)